english/new-years/new-years.md

```diff
@@ -1,5 +1,7 @@
-Ah . . . New Year's Eve . . . it all comes back to you now . . . sort of. 
+Ah . . . New Year's Eve . . . it all comes back to you now . . . sort of.
 It was a costume party, and, come to think of it, the Chief Marshmallow was there too.
 
-He was obviously angry about something. 
+He was obviously angry about something.
 It could be the horrible smell that was coming from the [raccoon in the corner.](../raccoon-corner/raccoon-corner.md)
+
+The tree in Rockefeller Center was stiil lit, but close up, the needles were beginning to look dried out. The fireworks in Central Park reflected off the faces of the people hurrying home after the ball dropped.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.